Makefile.defs

```diff
@@ -911,7 +915,7 @@ LPEG_DYNLIB=$(OUTPUT_DIR)/rocks/lib/lua/5.1/lpeg.so
 
 
 CMAKE_THIRDPARTY_LIBS := kpvcrlib,nanosvg
-CMAKE_THIRDPARTY_LIBS := $(CMAKE_THIRDPARTY_LIBS),evernote-sdk-lua,luajit,lpeg,turbo
+CMAKE_THIRDPARTY_LIBS := $(CMAKE_THIRDPARTY_LIBS),evernote-sdk-lua,luajit,lpeg, lua-htmlparser,turbo
```

Comment:
My bad, the grouping makes more sense a few lines down over by lua-Spore and luasec. It was basically just the first line I came across for this thing that often trips up new contributors.